search/views.py

```diff
@@ -214,6 +219,11 @@ def course_discovery(request):
         }
         log.debug(unicode(invalid_err))
 
+    except QueryParseError as parse_error:
```

Comment:
Please add a test case for this.

Reply:
👍  https://github.com/edx/edx-search/pull/49/files#diff-264640494855ee8f793ad467e25fa524R356